make-srpm.sh

```diff
@@ -150,7 +150,7 @@ in background fully transparently.
 
 %package -n csmatch
 Summary: A compiler wrapper that runs Smatch in background
-# Requires: smatch  (not in Fedora yet)
+Requires: smatch
```

Comment:
I think this should be a soft dependency.  `csmock-plugin-smatch` uses statically linked `/usr/bin/csmatch` from the host but installs the `smatch` RPM package into the chroot.  So `smatch` does not need to be installed on the host for `csmock-plugin-smatch` to work.

Reply:
I did not know that.  In that case that dependency is completely redundant, right?

Reply:
Actually no, anyone can use `csmatch` without the `csmock` plug-in.

Reply:
Exactly.  The dependency is redundant for `csmock-plugin-smatch` but useful for potential users of bare `csmatch`.

Reply:
Addressed in 27a46d482f19cf5ed89dc3c302dbbcae8c86db09 using `Recommends:`.